config/config.toml.example

```diff
@@ -143,6 +143,11 @@ query-feedback-limit = 1024
 # row count in statistics of a table is greater than it.
 pseudo-estimate-ratio = 0.7
 
+# Auto analyze will run if the ratio between the modify count and
+# the row count of a table is greater than it. It should be a positive
+# float number, otherwise auto analyze won't run.
```

Comment:
how about adding "set to 0.0 to disable auto analyze" ?

Reply:
I think the last sentence implies this.

Reply:
It's better to explicitly specify this important feature other than imply it.